Assets/MirageXR/Services/IBM-Watson/DialogueService.cs

```diff
@@ -75,108 +87,137 @@ void Start()
 
     }
 
-    private IEnumerator CreateService()
+    public IEnumerator CreateService()
```

Comment:
split it into two different methods (for openAI and IBMWatson)

Reply:
I think we need to completely rewrite the whole structure - I don't think we want to keep it like this, it's even mixing two styles...